test-lib.sh

```diff
@@ -773,4 +773,10 @@ ct_check_image_availability() {
   fi
 }
 
+ct_check_latest_is() {
+    local latest_version=
+
+    latest_version=$(grep 'VERSIONS' Makefile | rev | cut -d ' ' -f 1 | rev )
+    python3 "./check_imagestream.py" "$latest_version" | return 1
```

Comment:
I guess you wanted to use `... || return 1` but it is not necessary either, because return value of the function is what the last command returns. So, I believe you can remove the `| return 1` entirely.

Reply:
Good catch